docs/workflows/build-pages-with-storybook.md

```diff
@@ -0,0 +1,133 @@
+  return (
+    <PageLayout user={user}>
+      <DocumentHeader document={document} />
+      <DocumentList documents={subdocuments} />
+    </PageLayout>
+  );
+}
+```
+
+In such cases it is natural to use [args composition](../writing-stories/args#args-composition) to build the stories for the page based on the stories of the sub-components:
+
+```js
+// your-page.story.js
+
+import React from 'react';
+import DocumentScreen from './DocumentScreen';
+
+import PageLayout from './PageLayout.stories';
+import DocumentHeader from './DocumentHeader.stories';
+import DocumentList from './DocumentList.stories';
+
+export default {
+  component: DocumentScreen,
+  title: 'DocumentScreen',
+}
+
+const Template = (args) => <DocumentScreen {...args} />;
+
+export const Simple = Template.bind({});
+Simple.args = {
+   user: PageLayout.Simple.user,
+   document: DocumentHeader.Simple.document,
+   subdocuments: DocumentList.Simple.documents,
+};
+```
+
+This approach is particularly useful when the various subcomponents export a complex list of different stories, which you can pick and choose to build realistic scenarios for your screen-level stories without repeating yourself. By reusing the data and taking a DRY philosophy, your story maintenance burden is minimal.
+
+### Mocking connected components
+
+Render a connected component in Storybook by mocking the network requests that it makes to fetch its data. There are various layers in which you can do that.
+
+#### Mocking providers
+
+If you are using a provider that supplies data via the context, you can wrap your story in a decorator that supplies a mocked version of that provider. For example, in the [Screens](https://www.learnstorybook.com/intro-to-storybook/react/en/screen/) chapter of Learn Storybook we mock a Redux provider with mock data.
+
+Additionally there may be addons that supply such providers and nice APIs to set the data they provide. For instance [`storybook-addon-apollo-client`](https://www.npmjs.com/package/storybook-addon-apollo-client) provides this API:
+
+```js
+// my-component-with-query.story.js
+
+import MyComponentThatHasAQuery, {
+  MyQuery,
+} from '../component-that-has-a-query';
+ 
+export const LoggedOut = () => <MyComponentThatHasAQuery />; 
+LoggedOut.parameters: {
+    apolloClient: {
+      mocks: [
+        { request: { query: MyQuery }, result: { data: { viewer: null } } },
+      ],
+    },
+ };
+```
+
+#### Mocking imports
+
+It is also possible to mock imports directly, similar to Jest, using webpack’s aliasing. This is useful if your component makes network requests directly.
+
+
+```js
+// .storybook/main.js
+module.exports = {
+  // your Storybook configuration
+
+  webpackFinal: config => {
+    config.resolve.alias.fetch = '../__mocks__/fetch.js'
+    return config;
+   }
+};
+```
+
+You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story.
```

Comment:
I'm going to leave in the `div` as a remainder so that we don't loose track of what's required. Once you have the pr up we can remove it. Sounds good?